rules-reviewed/eap7/eap6/hibernate4-xml.windup.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate4-xml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            This ruleset provides analysis of configuration migration to Hibernate 5.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.4.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
+        </dependencies>
+        <sourceTechnology id="hibernate" versionRange="[4,5)" />
+        <targetTechnology id="hibernate" versionRange="[5,)" />
+        <targetTechnology id="eap" versionRange="[7,)" />
```

Comment:
I think the tag here needs to be eap7 to be consistent with the other rulesets. We don't have full support for specifying the version in the command line parameters yet.